### DISCUSSION TITLE
Audio Issue / Stereo / NDI Audio Filter

### DISCUSSION THREAD

**Discussion by MajeoT** [DistroAV/DistroAV#799] [Q&A - Support]
The sound recieved by the stream rig via an NDI audio filter from a dedicated audio source on my music line (like a virtual cable) on the goxlr is distorted/crackly and somewhat bass boosted/gain boosted it sounds like, when the OBS audio settings are set to "Stereo" or anything that isn't "Mono" for some reason, it wasn't like this last week.

If I drop it to Mono in Settings > Audio > General > Channels there is no distortion at all, however it doesn't sound as good as if it's on stereo though I doubt anyone would realistically notice as its more for background noise. None of the other settings, 2.1, 5.1 ect work, they're all crackly too though a little louder/clearer than Stereo. Changing it to mono is the "Advanced Audio Properties" but keeping it in Stereo as above still gives the same issue.

All of my sample rates are at 48kHz due to the GOXLR on both machines. I've tried reinstalling the ndi plugin but no luck.

I've tried "monitoring" the audio source on the game rig obs and it sounds fine, it sounds fine in the source its coming from (chrome/spotify) but it's disorted on the stream rig end when going out to the stream or using monitoring to test if its fixed or not, so something is happening when its being sent.

Running OBS version 28.1.2 64bit.

**Comment by MajeoT** [comment]
https://github.com/DDRBoxman/obs-ndi/releases/tag/4.10.1 Fixes this.

**Comment by Trouffman** [comment]
Fixed in official releases for couple versions. Thank you
